File: betamax-sample/src/main/java/software/betamax/android/sample/GithubService.java
package software.betamax.android.sample;

import java.util.List;

import io.reactivex.Observable;
import retrofit2.Response;
import retrofit2.http.GET;
import retrofit2.http.Path;

interface GithubService {
  @GET("/users/{owner}/repos") Observable<Response<List<Repository>>> repos(
      @Path("owner") String owner);
}

File: walkman-sample/src/main/kotlin/walkman/sample/GithubService.kt
package walkman.sample

import io.reactivex.Observable
import retrofit2.Response
import retrofit2.http.GET
import retrofit2.http.Path

internal interface GithubService {
  @GET("/users/{owner}/repos") fun repos(
      @Path("owner") owner: String): Observable<Response<List<Repository>>>
}
